kolibri/plugins/coach/assets/src/views/plan/CreateExamPage/ResourceSelection.vue

```diff
@@ -113,6 +113,7 @@
       const store = getCurrentInstance().proxy.$store;
       const route = computed(() => store.state.route);
       const topicId = computed(() => route.value.params.topic_id);
+      const bookMarksFlag = computed(() => route.value.query.bookmarks);
```

Comment:
I'd also love to see a comment here explaining what this is used for to give some context around how this whole solution works (ie, we're using the query parameter as a **"** route **"** of sorts to determine if we should be showing bookmarks or not.

Overall I think that @AlexVelezLl might do well to take inspiration from this approach for the work on #11786